GEMB/managelayers.m

```diff
@@ -35,19 +35,23 @@
 n=length(T);
 zY2=zY;

```

Comment:
If `zY2=zY` and both variables remain constant throughout the entire function, I think we can delete the `zY2=zY` line and just replace all instances of `zY2` with `zY`.